### DISCUSSION TITLE
AI寫論文

### DISCUSSION THREAD

**Discussion by Raywei0214** [1101-GS3073-C/Python-Community-GS3073-C#588] [General]
我覺得以後會普及，寫論文是非常花時間的，用AI就可以更有效率的表現出人類的想法，舉個例子來說，就像我現在正在打這篇文章，但事實上，或許可能交給AI來處理就不太會發現有如此那麼多的冗言贅字了
# https://news.ltn.com.tw/news/world/breakingnews/4144870

**Comment by Yilin20021016** [comment]
雖然說這樣很方便 但是後面出來的東西都是合體的?

**Comment by Valentinaa0214** [comment]
用 AI 來撰寫論文可能會產生一些道德或法律問題，例如抄襲或篡改等問題。
因此，在使用 AI 撰寫論文時需要遵循相關的道德規範和法律規定才行。
雖然 AI 可以作為幫助人們撰寫論文的工具，讓我們在過程中節省時間和精力，
但我認為它不能真正的完全替代人類的思考與創造力。
